lib/Optimizer/Transforms/LowerUnwind.cpp

```diff
@@ -323,7 +323,9 @@ struct ScopeOpPattern : public OpRewritePattern<cudaq::cc::ScopeOp> {
   LogicalResult matchAndRewrite(cudaq::cc::ScopeOp scope,
                                 PatternRewriter &rewriter) const override {
     auto iter = infoMap.opParentMap.find(scope.getOperation());
+    static_cast<void>(iter);
```

Comment:
Use `[[maybe_unused]]` rather than `(void)` casts as above.

Reply:
Agreed. Forgot about that - will apply.